Polish chat dialog

Thynix opened the pull request:
A button is less surprising to open a dialog than a link (should that be in the commit message?) and it's easier to tell what a button will do if it contains what it will do.

ArneBab commented:
I’ll still need to fix the indentation, clean the content and such, but the first step is done.

Thynix commented:
So do you want this merged at the moment? What do you think about running people's descriptions by them before publishing them?

Thynix commented on pages/en/people.php:
```diff
@@ -26,14 +30,90 @@
+
+<h5>Eleriseth</h5>
+
+<p>An anonymous contributor who provides pretty nice core-fixes over FMS. Went head-on-head with toad in autumn 2013 and took a break afterwards. Started again in2014. Named here, because his patches are great and got merged and because anonymous contributors are the central pillar we need to strengthen to make Freenet future proof if our political systems keep degrading at the current pace.
```
"in2014" needs a space.

Thynix commented on pages/en/people.php:
```diff
@@ -26,14 +30,90 @@
+</p>
+
+<h5>The folks from Frost</h5> 
+<p>Old-Time users, some with conspiracy theories about the developers, others who patch their nodes to be parasitic for the network. We have them to thank for the <a href="/USK@ZLwcSLwqpM1527Tw1YmnSiXgzITU0neHQ11Cyl0iLmk,f6FLo3TvsEijIcJq-X3BTjjtm0ErVZwAPO7AUd9V7lY,AQACAAE/fix-link-length/14/">link-length-fix</a>. If they had not broken the network this badly, we would likely not have dared to try ...
```
This strikes me as somewhat passive-aggressive to be front-and-center on the website. Do you still want it?

ArneBab commented:
I think it has to be cleaned up before it can be on the website. I just wanted to get the first step done (and recorded here) so I could clean up incrementally afterwards (and to ensure that if I should be hit by a bus, someone else would be able to continue).

Thynix committed:
help: Change "OK" button to "Open chat"

It's easier to anticipate what a button will do if it has an action on
it.

Thynix committed:
help: Style chat link as a button

Thynix committed:
help: bind to link CSS id instead of title

This is preferable because it's less surprising behavior, and is now
possible because the link is raw HTML instead of Markdown.
